gcp-ts-cloudrun/package.json

```diff
@@ -6,6 +6,7 @@
         "@types/node": "^12.0.0"
     },
     "dependencies": {
+        "@pulumi/awsx": "^0.40.0",
```

Comment:
Is there a reason for this seemingly-unrelated change? It also seems like a change we would not want to make.